lab5/code.cpp

```diff
@@ -0,0 +1,63 @@
+#include <fstream>
+
+int timesteps = 100000;
+const double dt = 0.01;
+const int c = 10; //how often will the coordinates be output
+const int ncol = 8;
+const int nstr = 2;
+const double a = 0.5; //the coefficient of the derivative of the zero order
+const double b = 0.01; //the coefficient of the first - order derivative
+double x_0 = 5, v_0 = 0, a_0 = -a*x_0 - b*v_0; //initial conditions
+const double coord_0[] = {x_0, v_0, a_0, x_0, v_0, a_0, 0, 0, 0}; //initial conditions
+
+//Data output
+void output(double coord[nstr][ncol], int time)
+{
+    std::ofstream foute;
+    foute.open("points.txt", std::ios::app);
+    foute << coord[1][0] << '\t' << coord[1][3] << '\t' << coord[1][1] << '\t' << coord[1][4] << '\t' << time*dt << std::endl;
+    foute.close();
+}
+
+//Euler's method
+double euler(double coord[nstr][ncol])
```

Comment:
What if we want to change the equation to solve? Say x' = x instead of x'' = -alpha*x